src/pages/cookie-policy/CookiePolicy.js

```diff
@@ -0,0 +1,33 @@
+import TitleWithDescription from '~/components/title-with-description/TitleWithDescription'
+import { Box, Container } from '@mui/material'
+import { useTranslation } from 'react-i18next'
```

Comment:
reorganise imports:
-libs
-components
-styles
and add empty spaces between them

Reply:
fixed